### PR TITLE
Update @ferocia-oss/osnap to fix issues with ES Modules

### LINKED ISSUE #447: Latest version of @ferocia-oss/osnap dependency broken
1.3.1 of @ferocia-oss/osnap is missing the src directory and causes loki to break. 

Was able to work around it for now by adding `"@ferocia-oss/osnap": "1.3.0",` to the resolutions key in my package.json

https://github.com/ferocia/osnap

### LINKED ISSUE #448: Issue with osnap when using pnpm
Hello, I recently upgraded to latest version of loki (0.28.1 -> 0.31.1) but I am having some issues when installing and running with pnpm: 
```
> loki

(node:59566) UnhandledPromiseRejectionWarning: Error: Cannot find module '@ferocia-oss/osnap/src/ios'
Require stack:
- loki/examples/react/node_modules/.pnpm/@loki+target-native-ios-simulator@0.31.0/node_modules/@loki/target-native-ios-simulator/src/create-ios-simulator-target.js
...
```
The dependency is missing the `src` folder
image

When I ran with yarn, I don't have the same issue. I have the following
image

I think it is an issue with [osnap](https://github.com/ferocia/osnap) but I could not file an issue on the GitHub project itself.

FYI, I'm using pnpm 7.30.0

Regards,
Pascal.

### PR DESCRIPTION
**Pull request by levibuzolic**
`@ferocia-oss/osnap` received some sub-dependency updates in 1.3.1 which resolved some outstanding security issues and CVEs, unfortunately some of these updates included some packages which have moved exclusively to ES Modules which broke in Common JS environments.

This has been fixed as of 1.3.5, the broken versions have been deprecated on NPM, however to help other developers avoid ending up with a bas version I figured it'd be good to bump the required version of `@ferocia-oss/osnap` in the `package.json`

Fixes #448
Fixes #447

### CODE REVIEW

File: packages/target-native-android-emulator/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   ],
   "main": "src/index.js",
   "dependencies": {
-    "@ferocia-oss/osnap": "^1.3.0",
+    "@ferocia-oss/osnap": "^1.3.5",
```
Comment (techeverri) [inline packages/target-native-android-emulator/package.json:23]: You also need to update the `yarn.lock` file. Run `yarn install` (or just `yarn`) in the root of the repo

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-v14.17.3
```
Comment (levibuzolic) [inline .nvmrc:1]: Needed to bump to the latest version of node 14 to support the current version of `execa`

Comment (techeverri) [reply .nvmrc:1]: The workflow "Integration tests: CLI" is failing with the error below

```
To run storybook you need to have node 16 or higher
```

https://github.com/oblador/loki/actions/runs/4655936027/jobs/8239071634?pr=450
image

Comment (techeverri) [reply .nvmrc:1]: I guess we need to use Node.js 16.x (or newer) here 👇 
https://github.com/oblador/loki/blob/7ca6aa0101151dd7ec56652708cdafd1083079cf/.github/workflows/tests.yml#L53

Comment (techeverri) [reply .nvmrc:1]: According to their official release schedule, Node.js 14.x will reach end-of-life this month

https://github.com/nodejs/release#release-schedule
image

Comment (techeverri) [reply .nvmrc:1]: Should we go ahead and remove support for Node.js 14.x (and older)? @oblador

Comment (oblador) [reply .nvmrc:1]: Let's!

Comment (levibuzolic) [reply .nvmrc:1]: #451

**Comment by levibuzolic** [comment]
Reauthrored and moved to #457

**Pull request closed without merging**